api/src/controllers/records.js

```diff
@@ -23,7 +24,10 @@ const process = (req, res, options) => {
   return auth.check(req, 'can_create_records')
     .then(() => generate(req, options))
     .then(doc => runTransitions(doc))
-    .then(result => res.json({ success: true, id: result.id }))
+    .then(result => {
+      messaging.send(result.id);
```

Comment:
Jumping the queue, ha?

Reply:
Yeah, I thought it'd make it just a little more responsive!